### PR TITLE
probe: add uuid to probe task

### PR DESCRIPTION
**Pull request by abonislawski**
One of schedule_ll PRs added uuid without changes in probes and probe build failed, this commit will fix it

Signed-off-by: Adrian Bonislawski <adrian.bonislawski@linux.intel.com>

### CODE REVIEW

File: src/probe/probe.c [hunk 1]
```diff
@@ -14,13 +14,18 @@
 #include <sof/lib/alloc.h>
 #include <sof/lib/dma.h>
```
Comment (dbaluta) [inline src/probe/probe.c]: @abonislawski please add the message you added in the PR description to this commit description. Commit description is kept in git history while PR will be lost in the infinity of github.

Comment (marc-hb) [reply src/probe/probe.c]: Also add the important "Fixes: 8b7275b65f17d6" reference in the commit message.

Comment (abonislawski) [reply src/probe/probe.c]: @dbaluta @marc-hb done

Comment (marc-hb) [reply src/probe/probe.c]: It's commit 0529c141f40b, my bad.  8b7275b is probably an earlier, not merged version (of the same commit)
Please use 12 characters `0529c141f40b` as recommended by `./scripts/checkpatch.pl --no-tree`

`checkpatch` isn't always right but these other `checkpatch` warnings seem easy to fix too.

Comment (abonislawski) [reply src/probe/probe.c]: Ok fine, updated

Comment (dbaluta) [reply src/probe/probe.c]: @abonislawski correct way of referencing commit is also specifying the commit subject.

e.g Fixes: COMMIT_HASH ("Commit subject")

Comment (dbaluta) [reply src/probe/probe.c]: @abonislawski also look at the checkpatch.pl warnings/errors. Are they addressable?

**Comment by lgirdwood** [comment]
@abonislawski best to fix checkpatch.

**Comment by abonislawski** [comment]
@lgirdwood @dbaluta all done

**Pull request merged**